src/axom/primal/geometry/BezierCurve.hpp

```diff
@@ -139,10 +107,7 @@ class BezierCurve
     const int sz = utilities::max(0, ord + 1);
     m_controlPoints.resize(sz);
 
-    for(int p = 0; p <= ord; ++p)
-    {
-      m_controlPoints[p] = pts[p];
-    }
+    for(int p = 0; p <= ord; ++p) m_controlPoints[p] = pts[p];
```

Comment:
```suggestion
    for(int p = 0; p <= ord; ++p) {
      m_controlPoints[p] = pts[p];
    }
```

I believe our style guide requires this.

Reply:
I'm not sure that if it does, although I agree that we probably want it to.

If we want to enforce curly braces for one-lines, we should probably update our clang-format rules to do this automatically in a separate PR.

Reply:
Never mind -- it's right at the top of our code formatting -- 
> 9.1 Curly braces should be used in all conditionals, loops, etc. even when the content inside the braces is a “one-liner”.
https://axom.readthedocs.io/en/develop/docs/sphinx/coding_guide/sec09_format.html

We should modify our clang-format to enforce this.

Reply:
That would be good, if it's supported.

BTW, thanks for confirming. I thought I may be hallucinating.

Reply:
Looks like it was added in clang@15 and we're on clang@10 for our `clang-format`, but we could use clang-tidy in the meantime.

See: https://stackoverflow.com/questions/26111162/can-clang-format-add-braces-to-single-line-if-statements-etc

Reply:
Unfortunately, the latest clang on any LC system is clang@14.0.6 (TOSS4).

Reply:
I'll admit, I did not know that was part of the style guide. I'll go through and make all the changes I see...

Reply:
No worries. While the curly braces are not necessary, they do make the code easier to understand, arguably. In the future, we will add this to our code formatter.

Reply:
I created an issue for this: https://github.com/LLNL/axom/issues/1117